.prettierignore

```diff
@@ -0,0 +1,9 @@
+.prettierignore
+package.json
+package-lock.json
+README.md
```

Comment:
Not sure about this - it's common for there to be `js/jsx` code blocks that would benefit from this. Any particular reason to not include?

Reply:
package and package-lock are generated by node every time you install or uninstall and that generator doesn't follow the prettier rules. You'll end up with unnecessary changes being committed. The ignore file doesn't need Prettier because there's no formatting in it.

If everyone is fine with what Prettier does to README.md files, you can turn that ignore off. You should definitely try it out for yourself before deciding. And while I have that limited to README.md there, you might end up wanting to add CONTRIBUTING.md, etc, after you see what it does, or you might like it and we can remove it entirely.

Reply:
I am for prettier if it only lints Javascript/JSX code blocks inside Markdowns

Reply:
I've tried prettier in READMEs and it only lints javascript blocks. However, code blocks in READMEs are often copied code and meant to be verbatim, so automatically re-formatting might not be desired.

Reply:
Linting `js/jsx` blocks in markdown is important, formatting it can be undesired as @enuchi mentioned. Maybe `*.md` is better suited?